Google.PowerShell/BigQuery/BqDataset.cs

```diff
@@ -192,6 +191,21 @@ private Dataset DoGetRequest(string project, string dataset)
                 return null;
             }
         }
+
+        /// <summary>
+        /// Helper function to take an array of key:value filters and turn it into a filter string.
+        /// </summary>
+        /// <returns></returns>
+        public string MakeFilter()
+        {
+            if (Filter == null) { return null; }
+            string labels = "";
+            foreach (string f in Filter)
```

Comment:
So this actually doesn't work.  It looked nice and compiled, but that misses out on the addition of "label." to the beginning of each term.  

Thought about doing `string.Join(" ", Filter)`, but that would end up with 
"test.one labels.other.two" when we need "labels.test.one labels.other.two"

Just going to revert to the original solution.